Mage/src/main/java/mage/cards/CardImpl.java

```diff
@@ -747,6 +753,9 @@ public boolean addCounters(Counter counter, UUID playerAddingCounters, Ability s
                 GameEvent addedAllEvent = GameEvent.getEvent(GameEvent.EventType.COUNTERS_ADDED, objectId, source, playerAddingCounters, counter.getName(), amount);
                 addedAllEvent.setFlag(isEffectFlag);
                 game.fireEvent(addedAllEvent);
+                if (informPlayers && !game.isSimulation()) {
```

Comment:
Similar to the `.informPlayers` comment, thoughts on moving `!game.isSimulation()` inside `getAddRemoveCountersLogMessage` so that it isn't accidentally forgotten?

Reply:
you never need the simulation check, it’s already built into the inform function but there are a lot of instances of it being used anyway

Reply:
> you never need the simulation check, it’s already built into the inform function but there are a lot of instances of it being used anyway

even better

Reply:
Good catch.  Removed the simulation checks.